### PR TITLE
Register windowsed.is-a.dev

### PR DESCRIPTION
**Pull request by windowsedd**
Register windowsed.is-a.dev with TXT record pointing to vc-domain-verify=windowsed.is-a.dev,e5344d1f26e8286585de.
hosted on vercel , its require using txt.

### CODE REVIEW

File: domains/windowsed.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "windowsed1225",
+           "email": "me@windowsed.me",
+           "discord": "906415095473655810",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.Wu2s52VXs8ZJdqRj-2BQR7uW8Y5PTkyDREXRM18p7MHsubvopdIV4SfcNaXFcYvjiyJBUeq-txAqRnlebZU_Z1RgeoN0raa50fEo1jyiXEngDzvnDcnqlWUeI49eXP-2JSbUEXm3Y_sX2tmjZvlyUg0gN1rNa7-p13d2WJB1yWG-DEQK680lyJ63q_3ImzjkzzwJaiNtqInvbro6IOI0zOGREvoycbSFB7dICeTodq5BNYm5coBrY1g_QxmhL5x6uVovN64SJssy1MAKhr8i2iR3KhfJkHWFIiaJviGfJqJic4w3TKIFPlvelclyqMDFatFdaR_cNSEQZNZrQLAafQ.MDwq2TyPR64xGApOjOEllQ.7sfYfncrWm11i5oFs_I5-kNFptjdv6vuqksAgUwBWJpSpY3vytpHY9Sm6vRT1E0K1A7nMlTcPFa5mYm-EuxkqigLAis2uqK3E0ydYXZ75kABX0JT7Qg8r3U10XGpjeKL0-5WnKRvytun9O7XJGuLPQ.-Q_dogK2zHsV1DuGhhaF-A"
+        },
+    
+        "record": {
+            "TXT": "vc-domain-verify=windowsed.is-a.dev,e5344d1f26e8286585de"
```
Comment (DEV-DIBSTER) [inline domains/windowsed.json:10]: This is a verification record that should be placed under a _vercel domain file.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/windowsed.json (hunk 1 above)
Comment (windowsed1225) [reply domains/windowsed.json]: i resolve that.

**Review by windowsedd — COMMENTED** [review]
i added discord connection txt string.

File: domains/windowsed.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "windowsed1225",
+           "email": "",
+           "discord": "906415095473655810",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.Efg0a1LhVERwlpp5SUaWr5ukKz0MizlImwqjnwtiHc8JCFpwy-U-b3nHiBHO1rcoT43tX5mpoBhBLl9BLNYVla6CRgtJngqAS1bajQaKP-oLRFwWp8EOLySOSR0Nm3yT82jl1ULJ-cPQF1mHtouGs3h6p0WIRtrNMc3FTumTGQft7e6tRqt5cgCjh9BukSb1wWCuBhqOz8EgS3s4zQ7YSamGHXrvX2qozbtJ5RX6wzOzq0EaKurOlZTqXY1l1o477ugTfiuZWidV0EwtT9MW40iHQmAhqLxF7MyK19fcihnzs7_Jk2rM-x_xQHLDtf69_vLxv9FAhVKPu1c1r0KOYg.Egzhvpxnuw5q5QCz3eaa-w.uWoVGC8o5fWpHxRKAc-Lez_OWuq5bQOVTzP7F1jmSgZpljr6ra2W_kyEgF2cl6bAaUMSHvo8-MqBN0g2koeG3kEyUZfpHBVrldcquIEQRHAzl1ZspSdRp8m3iJu5EwdMTxegTYeQ6k6zsW-RUNVTlg.ZxqJ8HZ6DoWHVW7goowohg"
+        },
+    
+        "record": {
+            "TXT": "vc-domain-verify=windowsed.is-a.dev,e5344d1f26e8286585de"
```
Comment (CuteDog5695) [inline domains/windowsed.json:10]: Please replace the TXT record with either one of these from our Vercel docs: https://www.is-a.dev/docs/vercel/

**Review by iostpa — DISMISSED** [review]

File: domains/windowsed.json (hunk 2 above)
Comment (windowsed1225) [reply domains/windowsed.json:10]: image
huh

Comment (CuteDog5695) [reply domains/windowsed.json]: ....
Thats only to **verify** your domain, which you already have `_vercel.windowsed.json`. Now you just need to replace the records with either a CNAME record of your website or a A record (but please give a preview of it)

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**